src/org/thoughtcrime/securesms/recipients/Recipient.java

```diff
@@ -194,11 +194,10 @@ public synchronized String toShortString() {
 
   public int getFallbackAvatarColor() {
     int rgb = 0x00808080;
-    if(dcContact!=null) {
-      rgb = dcContact.getColor();
-    }
-    else if(dcChat!=null){
+    if(dcChat != null) {
```

Comment:
isn't contact color equals the chat color of a 1:1 chat???

Reply:
_currently_, probably, yes. but it is better to request the semantically correct color from the core.